pkg/operator/upgradebackupcontroller/upgradebackupcontroller.go

```diff
@@ -325,3 +336,34 @@ func isBackupConditionExist(conditions []configv1.ClusterOperatorStatusCondition
 	}
 	return false
 }
+
+func getCurrentClusterVersion(cv *configv1.ClusterVersion) (string, error) {
```

Comment:
maybe a good addition to add into ceohelpers/common? 
https://github.com/openshift/cluster-etcd-operator/blob/master/pkg/operator/ceohelpers/common.go

Reply:
+1